config/log4j.properties

```diff
@@ -10,3 +10,9 @@ log4j.logger.org.apache.kafka=ERROR, stdout
 log4j.logger.org.I0Itec.zkclient=ERROR, stdout
 log4j.additivity.kafka.server=false
 log4j.additivity.kafka.consumer.ZookeeperConsumerConnector=false
+
+log4j.appender.file=org.apache.log4j.DailyRollingFileAppender
```

Comment:
+1 for rolling file appender.  I picked DFA because of consistency with other projects.  I'll change that